tests/testthat/test-renderer3-threeplot.R

```diff
@@ -0,0 +1,35 @@
+test_that("Simple rowspan layout works", {
+  plot_data <- data.frame(x = 1:3, y = c(2, 4, 6))
+
+  plot_collection <- list(
+    LeftPlot = ggplot(plot_data, aes(x, y)) +
+      geom_point() +
+      ggtitle("left_plot") +
+      theme_animint(rowspan = 2),
+
```

Comment:
please undo addition of empty lines in the middle of function calls in R code too